keyboards/planck/keymaps/gitdrik/config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
Could you add a copyright header to this file, as well?

Reply:
Fixed.